python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -363,281 +564,1057 @@ def plotGalaxyColors(self, catalogs, filenamer, dataId):
+                               catColors("HSC-I", "HSC-Z", mags, decentGalaxies),
+                               catColors("HSC-Z", "HSC-Y", mags, decentGalaxies),
+                               decentStarsMag, decentGalaxiesMag,
+                               "i - z  [{0:s}]".format(fluxColStr),
+                               "z - y  [{0:s}]".format(fluxColStr), self.fluxFilter, fluxColStr,
+                               xRange=xRange, yRange=(yRange[0], yRange[1] + 0.2),
+                               magThreshold=prettyBrightThreshold, camera=camera, hscRun=hscRun,
+                               unitScale=self.unitScale)
+                colorColor4MagPlots(dataId, filenamer(dataId, description=nameStr, style="noFitMagBins"),
+                                    self.log, catColors("HSC-I", "HSC-Z", mags, decentStars),
+                                    catColors("HSC-Z", "HSC-Y", mags, decentStars),
+                                    catColors("HSC-I", "HSC-Z", mags, decentGalaxies),
+                                    catColors("HSC-Z", "HSC-Y", mags, decentGalaxies),
+                                    decentStarsMag, decentGalaxiesMag,
+                                    "i - z  [{0:s}]".format(fluxColStr),
+                                    "z - y  [{0:s}]".format(fluxColStr), self.fluxFilter, fluxColStr,
+                                    xRange=xRange, yRange=(yRange[0], yRange[1] + 0.2),
+                                    magThreshold=prettyBrightThreshold, camera=camera, hscRun=hscRun,
+                                    unitScale=self.unitScale)
+            shortName = filtersStr + "Distance" + fluxColStr
             self.log.info("shortName = {:s}".format(shortName))
-            self.AnalysisClass(combined, ColorColorDistance("i", "z", "y", poly), "izyPerp", shortName,
+            self.AnalysisClass(combined, ColorColorDistance("i", "z", "y", poly, unitScale=self.unitScale,
+                                                            fitLineUpper=fitLineUpper,
+                                                            fitLineLower=fitLineLower),
+                               filtersStr + "Distance [%s] (%s)" % (fluxColStr, unitStr), shortName,
                                self.config.analysis, flags=["qaBad_flag"], qMin=-0.1, qMax=0.1,
-                               labeller=NumStarLabeller(len(catalogs)),
+                               magThreshold=prettyBrightThreshold, labeller=NumStarLabeller(2),
                                ).plotAll(dataId, filenamer, self.log,
-                                         Enforcer(requireLess={"star": {"stdev": 0.02}}), camera=camera,
-                                         tractInfo=tractInfo, patchList=patchList, hscRun=hscRun)
+                                         Enforcer(requireLess={"star": {"stdev": 0.03*self.unitScale}}),
+                                                  camera=camera, tractInfo=tractInfo, patchList=patchList,
```

Comment:
camera and hscRun are arguments of plotAll and should line up with the E in Enforcer.